oddbird/oddleventy
# [event-dates] Rethink how we show event post dates

![pascal-muller-4EajIuUxgAQ-unsplash](https://user-images.githubusercontent.com/1581694/114653029-c5b1bd80-9cac-11eb-805d-66547a51d28a.jpg)

## Trello
https://trello.com/c/C4AKsJpG/125-its-confusing-that-talks-are-listed-at-their-most-recent-date-but-show-the-created-date-on-the-resources-page

- [x] Remove "published date" on resources talk posts
- [x] Display 1-2 past talk dates as well (only upcoming are showing ATM)

## To test
https://deploy-preview-139--oddleventy.netlify.app/blog/

View upcoming and past "Talk" posts on /blog page to see the removed published date and a list of upcoming and/or past talk dates. 

![image](https://user-images.githubusercontent.com/1581694/114764276-5d54f180-9d29-11eb-800c-ab91243519fc.png)
